docs/UserGuide.md

```diff
@@ -163,15 +163,23 @@ Parameters:
 
 <div markdown="span" class="alert alert-primary">
   :exclamation: <b>Important:</b>
-Some commands are not available while on profile view page. 
+Only specific commands are available in the profile view page. 
 The list of available commands are: 
 <ul>
   <li> <code>help</code> </li>
-  <li> <code>edit</code> </li>
+  <li> <code>save</code> </li>
```

Comment:
did you check if the formatting is correct now, because I vaguely remember last week the website was looking off for this part (refer to the pic I sent last week)